reframe/core/pipeline.py

```diff
@@ -798,10 +798,16 @@ def __getattribute__(self, name):
         return super().__getattribute__(name)
 
     @classmethod
-    def __init_subclass__(cls, *, special=False, pin_prefix=False, **kwargs):
+    def __init_subclass__(cls, *, special=False, pin_prefix=False,
+                          required_version=[], **kwargs):
```

Comment:
Don't use mutable objects as default arguments...

```suggestion
                          required_version=None, **kwargs):
```